### PR TITLE
Polish translation of Tour of Scala: Mixins 

### PR DESCRIPTION
**Pull request by kamilduda**
Update mixins translation based on English versions

### CODE REVIEW

**Comment by SethTisue** [comment]
review by @jstefanska?

File: _pl/tour/mixin-class-composition.md [hunk 1]
```diff
@@ -1,20 +1,43 @@
...
+
+val d = new D
+println(d.message)  // wyświetli "Jestem instancją klasy B"
+println(d.loudMessage)  // wyświetli "JESTEM INSTANCJĄ KLASY B"
+```
+{% endscalafiddle %}
+
+Klasa `D` posiada nadklasę `B` oraz domieszkę `C`.
+Klasy mogą mieć tylko jedną nadklasę, ale wiele domieszek (używając kolejno słów kluczowych `extends`, a następnie `with`).
+Domieszki i nadklasy mogę posiadać tą samą nadklasę (typ bazowy).
```
Comment (jstefanska) [inline _pl/tour/mixin-class-composition.md]:
```suggestion
Domieszki i nadklasy mogą posiadać tą samą nadklasę (typ bazowy).
```

File: _pl/tour/mixin-class-composition.md [hunk 2]
```diff
@@ -24,35 +47,44 @@ abstract class AbsIterator {
...
+  class RichStringIter extends StringIterator("Scala") with RichIterator
+  val richStringIter = new RichStringIter
+  richStringIter foreach println
 }
 ```
 
-Klasa `iter` w funkcji `main` jest skonstruowana wykorzystując kompozycję domieszek `StringIterator` oraz `RichIterator` z użyciem słowa kluczowego `with`. Pierwszy rodzic jest nazywany _klasą bazową_ `Iter`, podczas gdy drugi (i każdy kolejny) rodzic jest nazywany _domieszką_.
+Nowo powstała `RichStringIter` posiada `StringIterator` jako nadklasę oraz `RichIterator` jako domieszkę.
+
+Mając do dyspozycji jedynie pojedyncze dziedziczenie nie byli byśmy w stanie osiągnąć takiego stopnia elastyczności.
```
Comment (jstefanska) [inline _pl/tour/mixin-class-composition.md]:
```suggestion
Mając do dyspozycji jedynie pojedyncze dziedziczenie, nie byli byśmy w stanie osiągnąć takiego stopnia elastyczności.
```

**Comment by SethTisue** [comment]
thank you @kamilduda and @jstefanska!!

**Pull request merged**